yar-api/pom.xml

```diff
@@ -60,7 +60,8 @@
                 <artifactId>maven-bundle-plugin</artifactId>
                 <configuration>
                     <instructions>
-                        <Import-Package>com.google.common.*;version=${guava.version},*</Import-Package>
+                        <Import-Package>com.google.common.*;version=${guava.version},javax.annotation;resolution:=optional,*</Import-Package>
+                        <_noee>true</_noee>
```

Comment:
Maven-bundle-plugin is not ready, this workaround avoids osgi.ee=UNKNOWN in manifest file causing errors at runtime